### DISCUSSION TITLE
One of the best themes for kanboard perhaps the best one currently

### DISCUSSION THREAD

**Discussion by Abraka** [JustFxDev/ThemeMaestro#12] [General]
Hands down man, after trying few themes for kanboard, I can say that yours is propably the best one. Very good organized, clear, not hurting my eyes, perfect visibility. Also color contrast of elements is great. So thx and keep up good work there pal!

**Comment by JustFxDev** [comment]
Thanks for the flowers 😄 

Be sure to get the latest code zip (not available as release yet) - There are some other glitches already fixed. e. g. Task input dialog is more tidy...

Next release is an ongoing task.

Cheers, Fx
